lib/commands/command-latest.bash

```diff
@@ -5,18 +5,31 @@ latest_command() {
 
   local plugin_name=$1
   local query=$2
+  local plugin_path
 
   [[ -z $query ]] && query="$DEFAULT_QUERY"
 
+  plugin_path=$(get_plugin_path "$plugin_name")
+  check_if_plugin_exists "$plugin_name"
+
   local versions
-  # pattern from xxenv-latest (https://github.com/momo-lab/xxenv-latest)
-  versions=$(asdf list-all "$plugin_name" "$query" |
-    grep -vE "(^Available versions:|-src|-dev|-latest|-stm|[-\\.]rc|-alpha|-beta|[-\\.]pre|-next|(a|b|c)[0-9]+|snapshot|master)" |
-    sed 's/^\s\+//' |
-    tail -1)
-
-  if [ -z "${versions}" ]; then
-    exit 1
+
+  if [ -f "${plugin_path}/bin/latest-stable" ]; then
+    versions=$(bash "${plugin_path}"/bin/latest-stable "$query")
+    if [ -z "${versions}" ]; then
+      # this branch requires this print to mimic the error from the list-all branch
+      printf "No compatible versions available (%s %s)\n" "$plugin_name" "$query" >&2
+      exit 1
+    fi
+  else
+    # pattern from xxenv-latest (https://github.com/momo-lab/xxenv-latest)
+    versions=$(asdf list-all "$plugin_name" "$query" |
+      grep -vE "(^Available versions:|-src|-dev|-latest|-stm|[-\\.]rc|-alpha|-beta|[-\\.]pre|-next|(a|b|c)[0-9]+|snapshot|master)" |
```

Comment:
I know this isn't a new change, but I think `Available versions` could be left out of this list. Plugins should just be printing the list of available versions and nothing else. An `Available versions` heading is not what is specified in the docs.

Reply:
Makes sense. I decided to leave it as it was purely for backward compatibility, just in case someone relied on it.